modules/components/pages/outputs/schema_registry.adoc

```diff
@@ -45,6 +45,9 @@ output:
   schema_registry:
     url: "" # No default (required)
     subject: "" # No default (required)
+    backfill_dependencies: true
+    translate_ids: true
```

Comment:
Should this be `translate_schema_ids` to match the field in the `redpanda_migrator`?